### PR TITLE
added leaflet map with dummy markers

### PR DESCRIPTION
**Pull request by soleil00**
added leaflet map with dummy markers
Screenshot 2024-05-04 at 20 19 34

### CODE REVIEW

**Comment by omotoshoayomikun** [comment]
While reviewing your code, I noticed that you're using next/image with an external image link in the MapPopupCard.tsx component without setting up a route in the next.config.mgs. This will cause errors. However, you can easily resolve this by adding the following code inside the nextConfig object in next.config.mjs:

const nextConfig = {
  images: {
    remotePatterns: [
      {
        protocol: 'https',
        hostname: "www.tse3.mm.bing.net" // the image domain
      }
    ]
  }
};

Adding this configuration should fix the issue.

**Comment by swoocn** [comment]
Hey @soleil00 - the following is what I am getting in the terminal when I run your changes locally using `npm run dev` ->

image

**Comment by swoocn** [comment]
This may be inline with what omotosho mentioned ->

image

File: constants/cooardinates.ts
Comment (swoocn) [inline constants/cooardinates.ts:1]: Could you correct the filename typo to **coordinates.ts**?

File: src/components/mappopup/MapPopupCard.tsx
Comment (swoocn) [inline src/components/mappopup/MapPopupCard.tsx:1]: Thoughts to make the folder/ package name more generic from /mappopup to /popups?  That way the package can be reused.

Comment (swoocn) [reply src/components/mappopup/MapPopupCard.tsx:1]: It feels we should mod the folder structure /components and build subfolders according to the screen type i.e., homescreen, buyer, seller, etc.  Thoughts?

File: constants/cooardinates.ts
Comment (soleil00) [reply constants/cooardinates.ts:1]: yeah so for reference error regarding window i tried to fix issue but it won't work, currently it hs no issue on app bcz by default nextjs consider component as server component and server component doesn't have access to window so we hav to mark componet as client and i did but still error presists and even i have use dynamic import it worked for react-leaflet but for leaflet it won't

**Review by swoocn — APPROVED** [review]
Leaflet feature looks good and functions as expected after suggested fixes. 👍

**Pull request merged**